ansible/playbook/tasks/update.yml

```diff
@@ -70,13 +70,27 @@
       include_role:
         name: postgres
 
+    # Set forking type to 'simple'
+    - name: Configure systemd
+      when: not is_docker
+      copy:
+        src: supervisord.service
+        dest: /usr/lib/systemd/system/supervisord.service
+        mode: 0644
+
+    - name: Remove old supervisord service configuration
+      when: not is_docker
+      file:
+        path: /etc/systemd/system/supervisord.service
+        state: absent
+
     # Start the services
     - name: Supervisord start        | Start supervisord service for AMI/OVF
       when: not is_docker
-      service:
+      systemd:
```

Comment:
should it be separate new task? restart, but supervisord still should be started?(I remember only old arch, when update was also part of the image build, so it needed still be enabled and started on that stage).

Reply:
No, this will actually start `supervisord` in case it is stopped. It will also run `systemctl daemon-reload` so the the service can pick up the changes in the config from the previous step.

Reply:
Or do you mean to `enable` it first and then only start?

Reply:
Ok, I'll make it a separate step to be sure.

Reply:
>Or do you mean to enable it first and then only start?
I mean when we create the image, when we enable supervisord?

Not when image starts or updates

Reply:
No, it seems like it is the same as before, and it worked. So I'd be reluctant to change.
Added a prior step.